### PR TITLE
Use RPC helper everywhere

### PR DESCRIPTION
**Pull request by hauleth**
- **fix: add schema field to all migrations**
- **ft: use RPC helper function everywhere**
- **chore: bump version**

### CODE REVIEW

File: lib/supavisor/helpers.ex [hunk 1]
```diff
@@ -329,12 +329,12 @@ defmodule Supavisor.Helpers do
...
     try do
       :erpc.call(node, module, function, args, timeout)
-      |> case do
-        {:EXIT, _} = badrpc -> {:error, {:badrpc, badrpc}}
-        result -> result
-      end
     catch
-      kind, reason -> {:error, {:badrpc, {kind, reason}}}
+      kind, reason ->
+        {:error, {:badrpc, {kind, reason}}}
```
Comment (abc3) [inline lib/supavisor/helpers.ex:334]:
```suggestion
      kind, reason -> {:error, {:badrpc, {kind, reason}}}
```
```suggestion
      kind, reason ->
        {:error, {:badrpc, {kind, reason}}}
```

File: lib/supavisor_web/ws_proxy.ex [hunk 2]
```diff
@@ -61,6 +61,6 @@ defmodule SupavisorWeb.WsProxy do
   @spec connect_local() :: {:ok, port()} | {:error, term()}
   defp connect_local() do
     proxy_port = Application.fetch_env!(:supavisor, :proxy_port_transaction)
-    :gen_tcp.connect('localhost', proxy_port, [:binary, packet: :raw, active: true])
+    :gen_tcp.connect(~c"localhost", proxy_port, [:binary, packet: :raw, active: true])
```
Comment (abc3) [inline lib/supavisor_web/ws_proxy.ex:64]: Is there any benefit to using sigils here?

Comment (abc3) [reply lib/supavisor_web/ws_proxy.ex:64]: ahh, it will be deprecated, right

Comment (hauleth) [reply lib/supavisor_web/ws_proxy.ex:64]: Yes, deprecation warnings.

File: lib/supavisor/helpers.ex (hunk 1 above)
Comment (hauleth) [reply lib/supavisor/helpers.ex:334]: I do not get this suggestions. You mean that it would be better to keep it in single line?

Comment (abc3) [reply lib/supavisor/helpers.ex:334]: yeah

Comment (abc3) [reply lib/supavisor/helpers.ex:334]: wdyt?

Comment (hauleth) [reply lib/supavisor/helpers.ex:334]: Can be, other entries are also in single line, so it will be more consistent. Will push updated commit in a moment.

**Review by abc3 — APPROVED** [review]

**Pull request merged**